packages/edit-site/src/components/sidebar-navigation-item/index.js

```diff
@@ -28,24 +28,23 @@ export default function SidebarNavigationItem( {
 			) }
 			{ ...props }
 		>
-			{ icon && (
-				<HStack justify="flex-start">
+			<HStack justify="flex-start">
+				{ icon && (
 					<Icon
 						style={ { fill: 'currentcolor' } }
 						icon={ icon }
 						size={ 24 }
 					/>
-					<FlexBlock>{ children }</FlexBlock>
-					{ withChevron && (
-						<Icon
-							style={ { fill: 'currentcolor' } }
-							icon={ chevronRight }
-							size={ 24 }
-						/>
-					) }
-				</HStack>
-			) }
-			{ ! icon && children }
+				) }
+				<FlexBlock>{ children }</FlexBlock>
+				{ withChevron && (
```

Comment:
Any possibility that this icon will be something other than a chevron? E.g., `+` or something?

Just asking. Looks easy enough to abstract should this ever be a use case in the future. 👍

Reply:
I would think it's unlikely, but nothing is written in stone!

Reply:
It's possible, although in the existing code it was called this, so I matched it. Pretty straightforward to change in the future if needed.